server/data.js

```diff
@@ -364,6 +372,10 @@ var async = require('async'),
 		return Data.isImported('_imported:_votes', _vid, callback);
 	};
 
+	Data.isVoteImported = function(_bid, callback) {
```

Comment:
shouldnt this be called `isBookmarkImported` ? 
(I know this whole repo is confusing, planning a major refactor right after NodeBB 1.0)